dashboard/config/routes.rb

```diff
@@ -393,6 +393,10 @@ module OPS
     post 'attend/:session_code/join', controller: 'workshop_enrollment', action: 'confirm_join_session'
     get 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'upgrade_account'
     post 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'confirm_upgrade_account'
+
+    get 'workshop_user_admin/facilitator_courses', controller: 'workshop_user_admin', action: 'facilitator_courses_form'
+    post 'workshop_user_admin/assign_course', controller: 'workshop_user_admin', action: 'assign_course'
+    get 'workshop_user_admin/remove_course', controller: 'workshop_user_admin', action: 'remove_course'
```

Comment:
`link_to 'Remove', {action: :remove_course, method: :delete, params: {course: course.course, facilitator_id: @facilitator.id}}`

evaluates to:

`<a href="/pd/workshop_user_admin/remove_course?course=CS+Fundamentals&amp;facilitator_id=5&amp;method=delete">Remove</a>`

instead of the intended result:

`<a data-method="delete" href="/pd/workshop_user_admin/remove_course?course=CS+Fundamentals&amp;facilitator_id=5" >Remove</a>`

So, I've left a TODO to figure out how to invoke `link_to` with `method: :delete` and also dynamically generate the `course` and `facilitator_id` query string parameters at a later date.